platform.py

```
@@ -14,223 +14,448 @@
 
 import os
 import contextlib
+import requests
```

Ruff (0.11.9)

`requests` imported but unused

Remove unused import: `requests`

(F401)